dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -79,6 +79,26 @@ Future<void> main() async {
         );
       }
 
+      section('Build ephemeral host app when SDK is on external disk');
+
+      // Don't clean, pretend the SDK was on an external drive with stray "._" files in the xcframework
```

Comment:
is this section dependent on running right after the previous section?

Reply:
No, I don't need the "Don't clean" part of this comment it doesn't make sense.
```suggestion
      // Pretend the SDK was on an external drive with stray "._" files in the xcframework.
```